src/App.js

```diff
@@ -1,19 +1,22 @@
 import "./App.css";
 import { Navbar } from "./components";
 import { Homepage, ProductListing, Wishlist, CartManagement, Login, Signup } from "./pages/index.js";
```

Comment:
You could have ignored writing /index.js over here, react automatically is able to get the path if you just put /pages and export from index.js file.